src/tables/cbdt.rs

```diff
@@ -1,12 +1,15 @@
 //! A [Color Bitmap Data Table](
 //! https://docs.microsoft.com/en-us/typography/opentype/spec/cbdt) implementation.
 
-use super::cblc::{self, BitmapFormat};
+use super::cblc;
+use crate::cblc::{Metrics, MetricsFormat, BitmapDataFormat};
```

Comment:
Why not `use crate::cblc::{self, ..}` ?

Reply:
Because apparently that's what my IDE does when it auto-imports things and I don't double-check it.